code/game/area/Space Station 13 areas.dm

```diff
@@ -58,37 +58,6 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 
 /*Adding a wizard area teleport list because motherfucking lag -- Urist*/
 /*I am far too lazy to make it a proper list of areas so I'll just make it run the usual telepot routine at the start of the game*/
-var/list/teleportlocs = list()
```

Comment:
These were removed so they can process in `SSmapping` for a few reasons:
1: Mapping lists should be generated inside the mapping SS
2: It was using unmanaged global lists, which arent good
3: This was called as soon as the server ticked out, adding to startup lag